lib/datasets/ames-housing.rb

```diff
@@ -0,0 +1,131 @@
+                        :neighborhood,
+                        :condition_1,
+                        :condition_2,
+                        :bldg_type,
+                        :house_style,
+                        :overall_qual,
+                        :overall_cond,
+                        :year_built,
+                        :year_remod_add,
+                        :roof_style,
+                        :roof_matl,
+                        :exterior_1st,
+                        :exterior_2nd,
+                        :mas_vnr_type,
+                        :mas_vnr_area,
+                        :exter_qual,
+                        :exter_cond,
+                        :foundation,
+                        :bsmt_qual, 
+                        :bsmt_cond,
+                        :bsmt_exposure,
+                        :bsmt_fin_type_1,
+                        :bsmt_fin_sf_1,
+                        :bsmt_fin_type_2,
+                        :bsmt_fin_sf_2,
+                        :bsmt_unf_sf,
+                        :total_bsmt_sf,
+                        :heating,
+                        :heating_qc,
+                        :central_air,
+                        :electrical,
+                        :first_flr_sf,
+                        :second_flr_sf,
+                        :low_qual,
+                        :fin_sf,
+                        :gr_liv_area,
+                        :bsmt_full_bath,
+                        :bsmt_half_bath,      
+                        :full_bath,
+                        :half_bath,
+                        :bedroom_abv_gr,
+                        :kitchen_abv_gr,
+                        :kitchen_qual,
+                        :tot_rms_abv_grd,
+                        :functional,
+                        :fireplaces,
+                        :fireplace_qu,
+                        :garage_type,
+                        :garage_yr_blt,
+                        :garage_finish,
+                        :garage_cars,
+                        :garage_area,
+                        :garage_qual,
+                        :garage_cond,
+                        :paved_drive,
+                        :wood_deck_sf,
+                        :open_porch_sf,
+                        :enclosed_porch,
+                        :three_ssn_porch,
+                        :screen_porch,
+                        :pool_area,
+                        :pool_qc,
+                        :fence,
+                        :misc_feature,
+                        :misc_val,
+                        :mo_sold, 
+                        :yr_sold,
+                        :sale_type,
+                        :sale_condition,
+                        :sale_price)
+      
+
+    def initialize
+      super()
+      @metadata.id = "ames-housing"
+      @metadata.name = "Ames Housing"
+      @metadata.url = "http://jse.amstat.org/v19n3/decock/DataDocumentation.txt"
+      @metadata.licenses = ["Unknown"]
```

Comment:
http://jse.amstat.org/jse_users.htm said:

> Unlike other American Statistical Association journals, the Journal of Statistics Education (JSE) does not require authors to transfer copyright for the published material to JSE. Authors maintain copyright of published material. Because copyright is not transferred from the author, permission to use materials published by JSE remains with the author. Therefore, to use published material from a JSE article the requesting person must get approval from the author.

Could you ask the author the license of this dataset?

Reply:
Still waiting for this. Will remind again beginning of December.

Reply:
I see!